sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -761,11 +761,11 @@ def create_or_update(
 
         :param entity: The resource to create or update.
         :type entity: typing.Union[~azure.ai.ml.entities.Job,
-            ~azure.ai.ml.entities.Model, ~azure.ai.ml.entities.Environment, ~azure.ai.ml.entities.Component,
-            ~azure.ai.ml.entities.Datastore, ~azure.ai.ml.entities.WorkspaceModelReference]
+             ~azure.ai.ml.entities.Model, ~azure.ai.ml.entities.Environment, ~azure.ai.ml.entities.Component,
```

Comment:
Is the new space before ~ intended?

Reply:
The new space is intended to be there to address the broken links issue. 
Screenshot showing missing space is resulting in unresolved entities:
![image](https://user-images.githubusercontent.com/20194110/211670411-93dd18a1-d403-4cc8-98de-8993d78d8ea5.png)

Reply:
yes, it's an attempt to resolve the references.